HOOKS.md

```diff
@@ -165,7 +159,7 @@ kubernetes:
 
 This hook will be executed *before* updating the Helm release with this binding context:
 
-```json
```

Comment:
it looks like we need to return `json` back here.

Reply:
It's a hack to ignore json's whining about "...".

Reply:
I'll put more spaces and it will look good.

Reply:
I get it.